config/initializers/cookie_rotator.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# As part of the Rails 7 upgrade, we need to convert legacy (SHA1) cookies to SHA256.
+# This can be removed after it has been in production for a little bit.
```

Comment:
Can you make an issue for doing this so we don't forget?

Reply:
Done! 
* https://otwarchive.atlassian.net/browse/AO3-6687
* https://otwarchive.atlassian.net/browse/AO3-6686 (for the config options because we'll want to clean those up too)